README.md

```diff
@@ -261,6 +261,7 @@ This section is its own hash, which should contain the following keys:
 * `listen`: these lines will be parsed and placed in the correct `frontend`/`backend` section as applicable; you can put lines which are the same for the frontend and backend here.
 * `backend_order`: optional: how backends should be ordered in the `backend` stanza. (default is shuffling). Setting to `asc` means sorting backends in ascending alphabetical order before generating stanza. `desc` means descending alphabetical order. `no_shuffle` means no shuffling or sorting.
 * `shared_frontend`: optional: haproxy configuration directives for a shared http frontend (see below)
+* `ignore_weights`: optional: stops haproxy backend 'weight' options being generated, even if the Nerve registrations contain this information. This will cause all backend servers to be treated equally by haproxy. This defaults to true so weights will *NOT* be used by default. 
```

Comment:
`ignore_weights` doesn't seem very useful as a setting and it is confusing that the default is shipped in a disabled state. There isn't a similar disable switch for `haproxy_server_options` and that's way more powerful than the weights setting (rhetorical question: why not just set `haproxy_server_options` to `weight 10`?). 

If the main concern is a breaking API change, then I would suggest bumping to a new major release for the next version. What do you all think?